packages/mui-material/src/Select/Select.js

```diff
@@ -14,10 +15,16 @@ import OutlinedInput from '../OutlinedInput';
 import useThemeProps from '../styles/useThemeProps';
 import useForkRef from '../utils/useForkRef';
 import styled, { rootShouldForwardProp } from '../styles/styled';
+import { getSelectUtilityClasses } from './selectClasses';
 
 const useUtilityClasses = (ownerState) => {
   const { classes } = ownerState;
-  return classes;
+
+  const slots = {
+    root: ['root'],
+  };
+
+  return composeClasses(slots, getSelectUtilityClasses, classes);
```

Comment:
When using `composeClasses`, all the classes will be under the `root` key. Do we know if this might have unintended consequences? For example, when forwarding classes [here](https://github.com/mui/material-ui/pull/38424/files#diff-3b8536ef0001d400b722b173fd9a64c9ce4eb8127bba26f7865977c5bb1a254fR122)?

Reply:
I think this might be the cause of the argos failure 🤔

Reply:
yup, you are right. argos got fixed after adding `select` and `icon` [here](https://github.com/mui/material-ui/pull/38424/files#diff-3b8536ef0001d400b722b173fd9a64c9ce4eb8127bba26f7865977c5bb1a254fR25-R26)

Reply:
I still think we might break missing keys in regards to [this line](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Select/Select.js#L115):

```ts
classes: inputProps ? deepmerge(classes, inputProps.classes) : classes,
```

For example, `"multiple"` classes will no longer be forwarded.

Could we add the missing classes and add a test for this? Checks are passing, but if I'm correct, they shouldn't be.

Reply:
All missing classes are actually applied in `SelectInput` component including `multiple` class [here](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Select/SelectInput.js#L91) . shouldn't these be enough 🤔 ? maybe that's why tests aren't failing

Reply:
Since all classes are in `SelectInput`, i've moved `root` class also to there.